src/pybind/mgr/dashboard_v2/services/ceph_service.py

```diff
@@ -53,8 +133,96 @@ def get_service(cls, service_name, service_id):
 
     @classmethod
     def get_pool_list(cls, application=None):
-        osd_map = cls.mgr.get('osd_map')
-        if not application:
-            return osd_map['pools']
-        return [pool for pool in osd_map['pools']
-                if application in pool.get('application_metadata', {})]
+        return _get_pool_list(cls.mgr, application)
+
+
+class CephDaemonMixin(object):
```

Comment:
I don't see this mixin being used anywhere. Also, the function exposed by this mixin is particular to the rbd-mirroring component, and therefore it should be called `CephRbdMirrorMixin` and expose all functions related to rbd-mirroring.